layouts/default.vue

```diff
@@ -4,6 +4,14 @@
       <v-btn icon nuxt to="/" class="mr-1">
         <v-icon>mdi-home</v-icon>
       </v-btn>
+
```

Comment:
Is this pending something or dead code?

Reply:
Yes, this is the button/link at the header for this page. I hide it for now as the rest functionality is already deployed on live